<ui version="4.0" >
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>174</height>
   </rect>
  </property>
  <property name="minimumSize" >
   <size>
    <width>280</width>
    <height>125</height>
   </size>
  </property>
  <property name="windowTitle" >
   <string>LOGIN</string>
  </property>
  <layout class="QGridLayout" >
   <item row="0" column="0" colspan="2" >
    <widget class="QFrame" name="frame" >
     <property name="frameShape" >
      <enum>QFrame::WinPanel</enum>
     </property>
     <property name="frameShadow" >
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" >
      <item rowspan="2" row="0" column="0" >
       <widget class="QFrame" name="frame_2" >
        <property name="frameShape" >
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow" >
         <enum>QFrame::Plain</enum>
        </property>
        <layout class="QGridLayout" >
         <item row="0" column="0" >
          <widget class="QLabel" name="label" >
           <property name="text" >
            <string>Login:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" >
          <widget class="QLineEdit" name="loginEdit" />
         </item>
         <item row="1" column="0" >
          <widget class="QLabel" name="label_2" >
           <property name="text" >
            <string>Hasło:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" >
          <widget class="QLineEdit" name="passwordEdit" >
           <property name="text" >
            <string/>
           </property>
           <property name="echoMode" >
            <enum>QLineEdit::Password</enum>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1" >
       <widget class="QPushButton" name="loginButton" >
        <property name="text" >
         <string>Zaloguj</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" >
       <widget class="QPushButton" name="newButton" >
        <property name="text" >
         <string>Nowy...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" >
    <widget class="QCheckBox" name="checkBox" >
     <property name="text" >
      <string>Pamiętaj hasło</string>
     </property>
     <property name="checked" >
      <bool>false</bool>
     </property>
     <property name="tristate" >
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="1" >
    <widget class="QCheckBox" name="checkBox_2" >
     <property name="enabled" >
      <bool>false</bool>
     </property>
     <property name="text" >
      <string>Loguj automatycznie</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
